src/puppetlabs/puppetdb/factsets.clj

```diff
@@ -52,11 +53,13 @@
   (zipmap (map (comp keyword :name) facts)
           (map :value facts)))
 
-(pls/defn-validated facts-expanded->wire-v5 :- {s/Keyword s/Any}
+(pls/defn-validated facts-expanded->wire-v5-classic :- {s/Keyword s/Any}
   [facts :- facts-expanded-query-schema]
   (facts-list-to-map
-   (map fact-query->wire-v5
-        (:data facts))))
+   (map fact-query->wire-v5 (:data facts))))
+
+(defn facts-expanded->wire-v5 [facts]
+  (facts-expanded->wire-v5-classic facts))
```

Comment:
What's the difference between classic and not classic here?

Reply:
I think that's detritus from the initial commits where both forms were supported. I'll consolidate them.